### ISSUE TITLE
Compilation error

### ISSUE THREAD

**Issue by guillemkub** [IvanVnucec/Optimal-REQUEST#47]
Hello.
When I try to compilate your code the next error jumps:
ERROR: Include directory to be added is not an include directory object.
I have installed the Meson builder and followed the steps.
Thanks you.

**Comment by IvanVnucec** [comment]
Can you provide an exact error message? I find it hard to think up a solution from your message only.
Also, can you provide what OS and what Meson version you have?

**Comment by IvanVnucec** [comment]
If you installed dependencies with the [install_deps.sh](https://github.com/IvanVnucec/Optimal-REQUEST/blob/master/scripts/install_deps.sh) script, installed programs will have different versions depending on Ubuntu version you might have because install_deps.sh script uses `apt-get` command.

By googling your issue, I've stumbled on [this](https://github.com/mesonbuild/meson/issues/6024) Issue thread, where they've concluded that the problem is in the Meson version. 

In [our CI server](https://github.com/IvanVnucec/Optimal-REQUEST/blob/master/.github/workflows/build_and_test_c.yml) configuration we use Ubuntu 20.04 to build Optimal request code and on Ubuntu 20.04 `apt-get install meson` command will install Meson 0.53.2 as stated [here](https://packages.ubuntu.com/focal/meson).

TLDR: I think that the issue is with the Meson version you have. Try using Meson 0.53.2 or newer.

**Comment by IvanVnucec** [comment]
i will close this issue
